RELEASE.md

```diff
@@ -1,16 +1,17 @@
-# Release 3.1.2:
+# Upcoming release:
 
 ## Major features and improvements
-- Promote viz from project specific commands to global commands.
-- Enable `kedro viz --load-file` outside of a Kedro project.
+- Promoted viz from project specific commands to global commands (#91)
+- Enabled `kedro viz --load-file` outside of a Kedro project (#91)
+- Added `--pipeline` option to visualize modular pipeline (#93)
+- Added `--env` option to pass configuration environment (#93)
 
 ## Bug fixes and other changes
 
 # Release 3.1.0:
 
 ## Major features and improvements
 
-**BREAKING CHANGE:**  Kedro<0.15.0 no longer works with this version of Kedro-Viz(#72)
```

Comment:
It was introduced in https://github.com/quantumblacklabs/kedro-viz/pull/72 (the line `catalog = get_project_context("create_catalog")(None)`), but when the PR was merged, we did not notice that it was breaking the compatibility with Kedro 0.14.*, and 3.1.0 was released (in Kedro 0.14.*, `create_catalog` needs an argument `conf`, but `None` was passed). 

So I added this line quickly 5 days ago https://github.com/quantumblacklabs/kedro-viz/pull/99 (and pinned kedro >=0.15.0). 

But I think you are right, I should not remove this line since we cannot undo the release 3.1.0. Instead I will say "the breaking change was fixed and it is not backward-compatible with Kedro 0.14.*" in the upcoming release note.